ib_insync/util.py

```diff
@@ -328,59 +329,54 @@ def timeRange(
         step (float): The number of seconds of each period
     """
     assert step > 0
-    start = _fillDate(start)
```

Comment:
The changes here and below are necessary because `mypy` doesn't know here that `start` can now only be `datetime.datetime` (it still thinks of it as the `Union`).